index.js

```diff
@@ -1,220 +1,220 @@
-
-    const filtered = applyFilter && !applyFilter(filter, m);
-    const limited = !Number.isNaN(+limit) && i >= limit;
-    if ((limited || filtered) && filterSummary) return null
-
-    bundleSize += m.size;
-    bundleOrigSize += m.origSize;
-
-    return (limited || filtered) ? null : m
-  }).filter((m) => m);
-
-  if (filterSummary) {
-    modules.forEach((m) => {
-      m.percent = Math.min(((m.size / bundleSize) * 100).toFixed(2), 100);
-    });
-  }
-
-  const bundleReduction = shakenPct(bundleSize, bundleOrigSize);
-  const moduleCount = (filterSummary ? modules : bundleModules).length;
-
-  return { bundleSize, bundleOrigSize, bundleReduction, modules, moduleCount }
-};
-
-const analyze = (bundle, opts) => new Promise((resolve, reject) => {
-  try {
-    const analysis = analyzer(bundle, opts);
-    return resolve(analysis)
-  } catch (ex) { return reject(ex) }
-});
-
-const formatted = (bundle, opts) => new Promise((resolve, reject) => {
-  try {
-    const analysis = analyzer(bundle, opts);
-    return resolve(reporter(analysis, opts))
-  } catch (ex) { return resolve(ex.toString()) }
-});
-
-const plugin = (opts = {}) => {
-  const writeTo = opts.writeTo || (opts.stdout ? console.log : console.error);
-
-  const onAnalysis = (analysis) => {
-    if (typeof opts.onAnalysis === 'function') opts.onAnalysis(analysis);
-    if (!opts.skipFormatted) writeTo(reporter(analysis, opts));
-  };
-
-  return {
-    name: 'rollup-plugin-analyzer',
-    buildStart: function () {
-      const ctx = this || {};
-      if (!ctx.meta || +(ctx.meta.rollupVersion || 0).charAt(0) < 1) {
-        const msg = '' +
-          `rollup-plugin-analyzer: Rollup version not supported\n${tab}` +
-          `Starting with 3.0.0 only Rollup >= 1.0.0 is supported\n${tab}` +
-          'Use rollup-plugin-analyzer 2.1.0 for prior Rollup versions';
-        console.error(msg);
-      }
-    },
-    generateBundle: function (outOpts, bundle, isWrite) {
-      const ctx = this || {};
-      if (!ctx.meta || +(ctx.meta.rollupVersion || 0).charAt(0) < 1) return null
-      const getDeps = (id) => {
-        return ctx.getModuleInfo ? ctx.getModuleInfo(id).importedIds : []
-      };
-
-      return new Promise((resolve, reject) => {
-        resolve();
-
-        const modules = [];
-        Object.entries(bundle).forEach(([outId, { modules: bundleMods }]) => {
-          bundleMods = bundleMods || {};
-          Object.entries(bundleMods).forEach(([id, moduleInfo]) => {
-            const dependencies = getDeps(id);
-            modules.push(Object.assign({}, moduleInfo, { id, dependencies }));
-          });
-        });
-
-        return analyze({ modules }, opts).then(onAnalysis).catch(console.error)
-      })
-    }
-  }
-};
-
-Object.assign(plugin, { plugin, analyze, formatted, reporter });
+const buf = ' ';
+const tab = '  ';
+const borderX = `${Array(30).join('-')}\n`;
+
+const formatBytes = (bytes) => {
+  if (bytes === 0) return '0 Byte'
+  const k = 1024;
```

Comment:
Weird..... all I changed is 1024.
Even if I choose "save without formatting" it still touches all these lines like it's removing some invisible character.